scripts/test_env.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""
+Setup or tear down a bare metal
+environment running a debug build of Iroha.
+"""
+
+import argparse
+import ipaddress
+import json
+import logging
+import os
+import pathlib
+import shutil
+import subprocess
+import sys
+
+
+class Network:
+    """
+    A network of bootstrapped peers to run on bare metal.
+    """
+    def __init__(self, setup_dir: pathlib.Path, host_addr: ipaddress.IPv4Address, n_peers: int = 4):
+        logging.info("Setting up test environment...")
+        self.setup_dir = setup_dir
+        self.n_peers = n_peers
+        peers_dir = setup_dir.joinpath("peers")
+        os.mkdir(peers_dir)
+        try:
+            shutil.copy2("./configs/peer/config.json", peers_dir)
+            shutil.copy2("./configs/peer/genesis.json", peers_dir)
+            shutil.copy2("./configs/peer/validator.wasm", peers_dir)
```

Comment:
I'd recommend making these configurable later down the line. If we run this script from a different location it should be possible to retrieve the project root.